e2e/vertical-full-page-map.spec.ts

```diff
@@ -0,0 +1,103 @@
+    await page.getByPlaceholder('Search for locations').fill('virginia');
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    await page.locator('#js-answersVerticalResults div').filter({ hasText: '1 Office Space 8.2 mi Close Card 7900 Westpark Drive Suite T200 McLean, VA 22102' }).nth(3).click();
+  });
+
+  test('clicking on a pin focuses on a result card', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').fill('virginia');
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    await page.getByRole('button', { name: 'Result number 5' }).click();
+    const locator = page.locator('#js-answersVerticalResults div').nth(134);
+    await expect(locator).toHaveClass(/pinFocused/);
+  });
+
+  // search when map moves works
+  test('search when map moves works', async ({ page }) => {
+    const searchLogo = '#js-yext-submit';
+    await page.mouse.move(600, 300);
+    await page.mouse.down();
+    await page.mouse.move(1200, 450, {steps: 5});
+    await page.mouse.up();
+    await page.waitForSelector(searchLogo, { state: 'detached' });
+    const detachedSearchLogo = await page.$(searchLogo);
+    expect(detachedSearchLogo).toBeFalsy();
+  });
+
+  test('search this area button works', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').fill('virginia');
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    // search bar yext logo remounts when search occurs
+    const searchLogo = '#js-yext-submit';
+    await page.getByLabel('Map controls').getByText('Search When Map Moves').click();
+    await page.locator('canvas').click();
+    await page.waitForSelector(searchLogo, { state: 'detached' });
+    const detachedSearchLogo = await page.$(searchLogo);
+    expect(detachedSearchLogo).toBeFalsy();
+  });
+
+  test('default initial search works and is enabled by default', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    const result = await page.locator('#js-answersVerticalResults div').nth(0);
+    await expect(result).toBeAttached();
+  });
+
+  test('pagination works', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').fill('virginia');
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    await page.getByLabel('Go to the next page of results').click();
+    const secondPage = page.locator('#js-answersVerticalResultsCount');
+    await expect(secondPage).toHaveText(/21/);
+  });
+
+  test('pagination scrolls the results to the top', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').fill('virginia');
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    await page.getByLabel('Go to the next page of results').click();
+    const isScrolledUp = await page.evaluate(() => {
+      return window.scrollY === 0;
+    });
```

Comment:
we should check the scrollTop of the `.Answers-resultsWrapper`. I don't believe the scrollY of the window itself changes on pagination

Reply:
I was struggling getting the correct component because Answers.resultsWrapper didn't seem to work/have a scrollTop member. Instead I just checked if the first element of the results was visible, but let me know if this is not robust enough for what we are testing here!